<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OYCZFilterServiceDialog</class>
 <widget class="QDialog" name="OYCZFilterServiceDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>572</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OYCZFilterServiceDialog</string>
  </property>
  <widget class="QComboBox" name="cmbObPath">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>40</y>
     <width>451</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>10</y>
     <width>261</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>请选择已知点数据：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>70</y>
     <width>441</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>请选择栅格生成范围：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>130</y>
     <width>221</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>请选择输出路径：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineOutputPath">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>160</y>
     <width>401</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="cmbRangePath">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>100</y>
     <width>451</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnBrowseOutput">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>160</y>
     <width>41</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cmbObVal">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>220</y>
     <width>451</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>190</y>
     <width>451</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>请选择插值属性字段：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnComfirm">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>440</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>确认</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCancel">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>440</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>250</y>
     <width>451</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>请输入栅格大小（与输入数据单位一致）：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>310</y>
     <width>451</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>（可选，默认15）请输入近邻点数：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineSize">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>280</y>
     <width>451</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineK_num">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>340</y>
     <width>451</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineC_val">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>400</y>
     <width>451</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>370</y>
     <width>451</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>（可选，默认0.3）请输入随机变化函数c值：</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
